plugin/storage/es/spanstore/service_operation.go

```diff
@@ -75,9 +76,10 @@ func (s *ServiceOperationStorage) Write(indexName string, jsonSpan *jModel.Span)
 		OperationName: jsonSpan.OperationName,
 	}
 	serviceID := fmt.Sprintf("%s|%s", service.ServiceName, service.OperationName)
+	hashedID := fmt.Sprintf("%x", sha256.Sum256([]byte(serviceID)))
 	cacheKey := fmt.Sprintf("%s:%s", indexName, serviceID)
```

Comment:
I don't see why we cannot use the hash for both document ID and cacheKey. Let's just have a single value.